### PR TITLE
Deleting test pdf created during confirm_certification test

### PR DESCRIPTION
**Pull request by mogrenAtWork**
#2397 

Fixes:
- Deleted form8-TEST.pdf created during confirm_certification.spec
- Adds .bybug_history file to .gitignore
- Changes the location of the FakePdfService to a tmp directory

### CODE REVIEW

**Comment by tejans24** [comment]
* I ran the tests multiple times, they seem to be passing consistently.
* Tests are properly breaking when code is changed

File: spec/support/fake_pdf_service.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@ def self.save_pdf_for!(form8)
   end
 
   def self.output_location_for(_form8)
-    File.join(Rails.root, "spec", "support", "form8-TEST.pdf")
+    File.join(Rails.root, "tmp", "pdfs", "form8-TEST.pdf")
```
Comment (tejans24) [inline spec/support/fake_pdf_service.rb]: Why is the path being changed here?

Comment (mogrenAtWork) [inline spec/support/fake_pdf_service.rb]: The fake_pdf_service should only be called when executing tests, we shouldn't save to a directory that contains permanent resources in this case. We should save it to the temp dir.

Comment (tejans24) [inline spec/support/fake_pdf_service.rb]: Sounds reasonable

**Pull request merged**